mesoshttp/client.py

```diff
@@ -713,3 +713,67 @@ def __register(self):
                 if line[count_bytes:]:
                     count_bytes = int(line[count_bytes:])
         return True
+
+    def combine_offers(self, offers, operations, options=None):
+        '''
+        Accept offers with task operations
+
+        :param offers: offers to be accepted
+        :type offers: list
+        :param operations: JSON TaskInfo instances to accept
+        :type operations: list of json TaskInfo
+        :param options: optional filters
+        :type options: JSON filters instances
+        '''
+        if not operations:
+            self.logger.debug('Mesos:Accept:no operation to accept')
+            return True
+
+        if not offers:
+            self.logger.debug('Mesos:Accept:no offers to accept')
+            return True
+
+        ids = [o.get_offer()['id']['value'] for o in offers]
+        offer_ids = [{'value': oid} for oid in ids]
+        self.logger.debug('Mesos:COMBINE Offer ids:' + ','.join(ids))
+
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+            'Mesos-Stream-Id': self.streamId
+        }
+
+        tasks = []
+        for operation in operations:
+            if 'agent_id' not in operation:
+                operation['agent_id'] = {
+                    'value': self.offer[0]['agent_id']['value']
```

Comment:
What is this self.offer here? I do not see any reference to this.
In client, this is no offer instance attribute

Reply:
You are right. I will fix it.